Atomic/atomic.py

```diff
@@ -771,6 +881,343 @@ def install(self):
         if not self.args.display:
             return util.check_call(cmd)
 
+    def _oci_images(self):
+        repo = self._get_ostree_repo()
+
+        prefix = "dockerimg/"
+
+        revs = [x for x in repo.list_refs()[1] if x.startswith(prefix) and len(x) != len(prefix) + 64]
+        max_column = max([len(rev) for rev in revs]) + 2
+        col_out = "{0:%d} {1:64}" % max_column
+        self.writeOut(col_out.format("IMAGE", "COMMIT"))
+
+        for rev in revs:
+            commit = repo.resolve_rev(rev, False)[1]
+            self.writeOut(col_out.format(rev, commit))
+
+    def systemctl_command(self, cmd, name):
+        cmd = self.sub_env_strings(self.gen_cmd(["systemctl", cmd, name]))
+        self.display(cmd)
+        if not self.args.display:
+            util.check_call(cmd, env=self.cmd_env())
+
+    def _uninstall_oci_container(self, name):
+        ocidir = os.path.realpath("/var/lib/containers/atomic/%s" % name)
+        self.args.display = False
+        try:
+            self.systemctl_command("stop", name)
+        except:
+            pass
+        try:
+            self.systemctl_command("disable", name)
+        except:
+            pass
+
+        if os.path.exists(os.path.join("/etc/systemd/system", "%s.service" % name)):
+            os.unlink(os.path.join("/etc/systemd/system", "%s.service" % name))
+        if os.path.exists("/var/lib/containers/atomic/%s" % name):
+            os.unlink("/var/lib/containers/atomic/%s" % name)
+        if os.path.exists("/var/lib/containers/atomic/%s.0" % name):
+            shutil.rmtree("/var/lib/containers/atomic/%s.0" % name)
+        if os.path.exists("/var/lib/containers/atomic/%s.1" % name):
+            shutil.rmtree("/var/lib/containers/atomic/%s.1" % name)
+
+    def _prune_ostree_images(self):
+        repo = self._get_ostree_repo()
+        refs = {}
+        app_refs = []
+        prefix = "dockerimg/"
+
```

Comment:
it is the prefix for OSTree branches generated from a Docker import.  I will define it in a constant to make it clearer